src/App.js

```diff
@@ -4,16 +4,24 @@ import AddTodoForm from './AddTodoForm';
 
 
 function App() {
+
  
     const [newTodo, setNewTodo] = React.useState('');
  
-
+    /*1.4 new state variable named todoList 
```

Comment:
I would delete this comment as this one seems to be only the instructions for the assignment.

Reply:
done